research/format.markdown

```diff
@@ -0,0 +1,124 @@
+)
+
+> Behaviour:  
+      >>[Before](https://json-schema.org/draft-02/draft-zyp-json-schema-02.txt): MAY validate.   
+      [After](https://json-schema.org/draft-04/draft-fge-json-schema-validation-00#rfc.section.7.2):  
+      a. they SHOULD implement validation for attributes defined below;  
+      b. they SHOULD offer an option to disable validation for this keyword.  
+      c. Implementations MAY add custom format attributes but not peer implementations.
+
+| Change Type | Format Attribute | Description                                                                                 |
+|-------------|------------------|---------------------------------------------------------------------------------------------|
+| Deleted     | `date`             | This SHOULD be a date in the format of YYYY-MM-DD                                          |
+| Deleted     | `time`             | This SHOULD be a time in the format of hh:mm:ss                                            |
+| Deleted     | `utc-millisec`     | This SHOULD be the difference, measured in milliseconds, between the specified time and midnight, 00:00 of January 1, 1970 UTC. The value SHOULD be a number (integer or float) |
+| Deleted     | `regex`            | A regular expression, following the regular expression specification from ECMA 262/Perl 5  |
+| Deleted     | `color`            | This is a CSS color (like `#FF0000` or `red`), based on [CSS 21](https://www.w3.org/TR/2007/CR-CSS21-20070719/)                          |
+| Deleted     | `style`            | This is a CSS style definition (like `color: red; backgroundcolor:#FFF`), based on [CSS 2.1](https://www.w3.org/TR/2007/CR-CSS21-20070719/) |
+| Deleted     | `phone`            | This SHOULD be a phone number (format MAY follow E.123)                                  |
+
+### Changes
+
+| Attribute   | Previous Description                                                                      | Present Description           |
+|-------------|-------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| `date-time`   | This SHOULD be a date in ISO 8601 format of YYYY-MMDDThh:mm:ssZ in UTC time. This is... | A string instance is valid against this attribute if it is a valid date representation as defined by RFC 3339, section 5.6[RFC3339]                           |
+| `URI`         | SHOULD be a uri                                                                           | A string instance is valid against this attribute if it is a valid URI, according to [RFC3986]          |
+| `email`       | SHOULD be email                                                                           | A string instance is valid against this attribute if it is a valid Internet email address as defined by RFC 5322, section 3.4.1 [RFC5322]                        |
```

Comment:
Same on this and on `ip-address`. Seems like clarifications on the spec but no actual differences in the strings they are supposed to describe/validate?

Reply:
So.. this isn't necessary ? that rfc is specified

Reply:
We care about semantic changes to the keyword. If its only an RFC clarification that doesn't impact the keyword, then let's leave it out

Reply:
Ohhk, then there are none of changes to be considered..